pylintrc

```diff
@@ -139,12 +139,13 @@ disable=print-statement,
         deprecated-sys-function,
         exception-escape,
         comprehension-escape,
-        invalid-name, 
-        bad-classmethod-argument, 
-        bad-mcs-classmethod-argument, 
+        invalid-name,
+        bad-classmethod-argument,
+        bad-mcs-classmethod-argument,
         no-self-argument,
         bad-continuation,
-        bad-whitespace
+        bad-whitespace,
+        unsubscriptable-object,           # https://github.com/PyCQA/pylint/issues/3882 broken in 3.9
```

Comment:
This is the important change in this file, the whitespace changes are part of my automatic IDE setting